tests/scripts/list-identifiers.sh

```diff
@@ -47,9 +47,9 @@ done
 
 if [ $INTERNAL ]
 then
-    HEADERS=$( ls include/mbedtls/*_internal.h library/*.h | egrep -v 'compat-1\.3\.h|bn_mul' )
+    HEADERS=$( ls library/*.h )
```

Comment:
Note: previously `bn_mul` was excluded (here and in the else branch). I'm not sure why that was (should have left a comment), so I'm not sure if it's OK to stop excluding it... Was that change intentional?

Note 2: I'm not mentioning `compat-1.3` as it's being removed by #4150

Reply:
I double-checked, and I think removing `bn_mul` here is OK.

Reply:
I thought it was being specifically included not excluded so no in that sense I thought the new statement would catch the same files as before (as bn_mul.h) is now internal in `library/`. However looking at it I'm not sure why it was ever excluded in the first place (git blames you: d1ddd29e81) as it's just defining a lot of macros. Is there a reason why it needs to be excluded from the list of identifiers?

Reply:
No, in the meantime I double-check and found no reason to exclude it: excluding it may make it easier to pass `check-names.sh`, but since it now passes without excluding it (as the CI results for this PR show), I don't think there's any reason to exclude it any more.

(Said otherwise, after looking into it a bit more I withdraw this point, but I didn't want to remove the comment in case you saw it in the meantime and got confused by disappearing comments. But I've marked the conversation as resolved.)